ddtrace/contrib/tornado/compat.py

```diff
@@ -1,9 +1,12 @@
-from ..util import require_modules
-
-
-optional_modules = ['concurrent.futures']
-
-with require_modules(optional_modules) as missing_modules:
+try:
     # detect if concurrent.futures is available as a Python
     # stdlib or Python 2.7 backport
-    futures_available = len(missing_modules) == 0
+    from ..futures import patch as wrap_futures, unpatch as unwrap_futures
+    futures_available = True
+except ImportError:
```

Comment:
Under which conditions could this happen/fail?
Would this instrumentation be partial if we use the dummy wrapper? Should we log something?

Reply:
It could happen if you're using `py27` without `futures`. In that case:
* we don't need to do anything because it will happen in the same thread
* we do partial instrumentation only if something else that we don't support is used

Reply:
Lovely!

Reply:
Lovely!